int32 surface_count

geometry_msgs/PointStamped[] centers

geometry_msgs/Vector3Stamped[] normals

float32[] areas

float32[] long_sides

float32[] short_sides

geometry_msgs/PolygonStamped[] corners

geometry_msgs/Vector3Stamped[] x_axes

geometry_msgs/Vector3Stamped[] y_axes

geometry_msgs/Vector3Stamped[] z_axes
